SU-SWS/stanford_profile_helper
# D8CORE-2040 Update styles for react paragraphs V2

# READY FOR REVIEW

# Summary
- Updated Drush command for reach paragraphs V2

# Need Review By (Date)
- 8/28

# Urgency
- Medium

# Steps to Test
1. checkout https://github.com/SU-SWS/acsf-cardinalsites/pull/185 & `composer install -n`
1. do a site install or `blt drupal:update`
1. run `drush stanford-profile:stress-test-components` to build a stress test page
1. view the page and see its greatness

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
